Gruntfile.coffee

```diff
@@ -39,6 +39,7 @@ module.exports = (grunt) ->
 			"sprites"
 			"css"
 			"js"
+			"string-replace"
```

Comment:
@nschonni so turns out that string-replace doesn't have multi-task support so I'm going to try to use https://github.com/yoniholmes/grunt-text-replace instead. I've been juggling a few things but I'll try to finish it up asap.